Add CP and EA flags

NotTsunami opened the pull request:
* CP uses France flag
* EA uses Spain flag

Defined in Emoji 1.0 spec, partially addresses #729

NotTsunami commented:
The spec is available [here](https://unicode.org/Public/emoji/1.0/emoji-data.txt).

lipis commented on css/flag-icon.css:
```diff
@@ -1512,6 +1512,24 @@
+  background-image: url(../flags/4x3/ea.svg);
+}
+.flag-icon-ea.flag-icon-squared {
+  background-image: url(../flags/1x1/aea.svg);
```
is that a typo? `aea.svg`

NotTsunami replied:
Yep, good catch. What's the overall view on the PR? I can push a fix if it's gonna move forward

lipis commented:
Let's do it!

KingDarBoja commented:
This PR is an awaited feature to be used in web browsers, as other emoji libraries does support the same unicode spec. 🚀

NotTsunami commented:
> This PR is an awaited feature to be used in web browsers, as other emoji libraries does support the same unicode spec. 🚀

Updated. Fixed an issue with the previous patchset where clip IDs were still not unique. Should be ready to merge.

antoinebrault commented:
any ETA on merging that PR?
